lib/views/pages/home/content/map/components/map_pin_pop_up.dart

```diff
@@ -0,0 +1,74 @@
+import "package:flutter/material.dart";
+
+class MapPinPopUp extends StatelessWidget {
+  //key for the arrow button
+  static const arrowButtonKey = Key("arrowButton");
+  const MapPinPopUp({
+    super.key,
+    this.title,
+    this.content,
+    this.navigationAction,
+    this.displayButton = false,
+  });
+
+  final String? title;
```

Comment:
Is it possible to have a pop-up without a title? I feel like this is a bit weird. I think it would be nice to make this parameter non-nullable, and use `required this.title` in the constructor.

Reply:
fixed with [791bcf4](https://github.com/ProximaEPFL/proxima/pull/335/commits/791bcf45e1f74336507c3693a62c575ce7bb3fe2)